src/Lib/Mail/Mail.php

```diff
@@ -0,0 +1,105 @@
+<?php
+declare(strict_types = 1);
+
+namespace App\Lib\Mail;
+
+class Mail
+{
+
+    /** @var string */
+    private $headders;
+
+
+    /**
+     *
+     * @param string $to
+     * @param string $from
+     * @param string $fromName
+     * @param string $content
+     */
+    public function __construct(string $to, string $from, string $fromName, string $subject, string $content)
+    {
+        $this->to = $to?$to:false;
+        $this->from = testEmail($from, 'From')?$from:false;
+        $this->fromName = $fromName?$fromName:false;
+        $this->subject = $subject?$subject:false;
+        $this->content = $content?$content:false;
+    }
+
+
+    /**
+    * Add hidden copy
+    *
+    * @return void
+    */
+    public static function addBcc($bcc):void
```

Comment:
Return $this, this will lead to chaining, like `$mail->addBcc('foo@bar.com')->send()`. Same addReplyTo method.